PSKoans/Public/Get-Blank.ps1

```diff
@@ -20,11 +20,11 @@
         https://github.com/vexx32/PSKoans
     #>
     [CmdletBinding()]
-    [OutputType([void])]
+    [OutputType([Blank])]
```

Comment:
Reference the type name as a string rather than the type object itself; it'll defer looking the type up until the function is called, by which time the type should definitely be in scope.
```suggestion
    [OutputType('Blank')]
```